pyproject.toml

```diff
@@ -124,5 +117,29 @@ plugins = ["pydantic.mypy"]
 [tool.pytest.ini_options]
 markers = [
     "integration: marks tests as integration (deselect with '-m \"not integration\"').",
-    "asyncio: marks tests as async."
+    "asyncio: marks tests as async.",
+]
+
+[tool.ruff]
+src = ["truss", "truss-chains", "truss-utils"]
+target-version = "py38"
+line-length = 88
+lint.extend-select = ["I"]
+lint.ignore = ["E402"]
```

Comment:
This was not being enforced previously, and since it generally can have runtime implications wanted to leave it alone for now.

Reply:
Could you please replace error codes with human-readable error names?